### DISCUSSION TITLE
Is it possible to use Firebase Firestore with Kobweb?

### DISCUSSION THREAD

**Discussion by o-nos** [varabyte/kobweb#576] [Q&A]
as simple as that :)
just need to know before setting Kobweb up

**Comment by bitspittle** [comment]
Yes it is possible. The hardest part is writing bindings for their APIs.

You may want to see if you can get https://github.com/GitLiveApp/firebase-kotlin-sdk working in your project. 

For learning, I worked on https://github.com/bitspittle/firebase-kotlin-bindings for myself, it's definitely not done nor meant for widespread use, but you can see a lot of examples of how I wrote bindings. My goal was to make an API that was strictly kotlin-idiomatic, which often involved some pretty thick wrappers around the underlying firebase JS APIs.
